src/source/documentUploader.ts

```diff
@@ -22,5 +23,8 @@ export async function uploadBatch(
 
     await create(platformClient, fields);
   }
-  return strategy.uploadBatch(batch);
+  const res = await strategy.upload(batch);
+  await strategy.postUpload();
```

Comment:
in relation to https://github.com/coveo/push-api-client.js/pull/117/files#r838077559,
```suggestion
  await strategy.postUpload?.();
```